### DISCUSSION TITLE
Adding Youtube link

### DISCUSSION THREAD

**Discussion by clarus** [upkie/upkie#479] [Ideas]
There are a few videos of Upkie on Youtube https://www.youtube.com/@upkie You could add links to it, if not already done!

Also, on the GitHub organization, it is possible to have a header page, such as on https://github.com/formal-land , to provide some information.

**Comment by stephane-caron** [comment]
Thank you for your suggestion :+1: Org readme created and link added in 5aa918f28409fa22b437d31c3c1ed7dc56cddadd.

**Reply by clarus** [reply]
![image](https://github.com/user-attachments/assets/203e4371-d9c6-4124-8fc4-ecf806cc14a9)
